lib/http/endpoint.js

```diff
@@ -218,15 +218,10 @@ const defaultErrorWriter = (error, request, response) => {
       details: error.problemDetails
     });
   } else {
-    const details = {};
-    if (error?.stack != null)
-      details.stack = error.stack.split('\n').map((x) => x.trim());
-
     debugger; // trip debugger if attached.
     process.stderr.write(inspect(error));
     response.status(500).type('application/json').send({
-      message: `Completely unhandled exception: ${error?.message}`,
```

Comment:
The discussion in the issue was about stack traces, but here the error message is also not being returned. That seems OK to me: the user will still be able to see it in their logs. But could you say more about why you made that change? Do you think that as with stack traces, secrets could end up in error messages?

Reply:
> Do you think that as with stack traces, secrets could end up in error messages?

Yes, this :+1: